utilities/run_unit_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+usage () {
+    echo -e "\tUSAGE: $0 [main|initial] [device|simulator]"
```

Comment:
the parameters are required. we can use angular brackets here.

```suggestion
    echo -e "\tUSAGE: $0 <main|initial> <device|simulator>"
```

Reply:
fixed